### PR TITLE
PXB-2706 encrypt-key example key is too short for AES256 (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M source/xtrabackup_bin/backup.encrypting.rst

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/backup.encrypting.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ encryption key. An encryption key can be generated with a command like
```
Comment (rahulmalik87) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/backup.encrypting.rst:26]: the command should be openssl rand -base64 32

**Review by rahulmalik87 — CHANGES_REQUESTED** [review]

**Review by altmannmarcelo — APPROVED** [review]
LGTM after addressing @rahulmalik87 request.

**Comment by altmannmarcelo** [comment]
Does it apply to 2.4 documentation too ?

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/backup.encrypting.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/backup.encrypting.rst:26]: Changed from 32

**Comment by rahulmalik87** [comment]
LGTM.

**Comment by patrickbirch** [comment]
This change does not apply to 2.4.

**Pull request merged**